TS3QueryLib.Net.Core/Server/Entitities/BanListEntry.cs

```diff
@@ -27,6 +28,7 @@ BanListEntry IEntity<BanListEntry>.ApplyFrom(CommandParameterGroup currentParame
 
             Id = currentParameterGroup.GetParameterValue<uint>("banid");
             Ip = currentParameterGroup.GetParameterValue("ip");
+            LastNickname = currentParameterGroup.GetParameterValue("LastNickname");
```

Comment:
Is it the real name of the parameter. Casing?

Reply:
Yes 
![img](https://user-images.githubusercontent.com/10272930/72989602-e1524f00-3dee-11ea-8cb6-aa938e616e14.png)